src/static/scss/components/_app-modal.scss

```diff
@@ -98,15 +113,24 @@
 
       text-align: left;
     }
+
+    @at-root .-onboarding#{&} {
+      overflow-y: unset;
+
+      max-width: 55rem;
```

Comment:
Maybe it's better to follow something like the document page (document-body.scss): max-width: 66ch.
Make reading easier 🤘